Bottom: c2abee6adb2199ed3a0086fe125157903ea636fd
Top:    2b0a0ac2409a9141407bb8272427530b332aca44
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 02:35:53 +0100

Refresh of use-heap-structure-to-find-hol

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 5a40972..174fd17 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1208,6 +1208,7 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
       all_pa_info[i].BoundingBox.Y2 = curc->contours->ymax;
       all_pa_info[i].pa = curc;
       r_insert_entry (tree, (const BoxType *) &all_pa_info[i], 0);
+      i++;
     }
   while ((curc = curc->f) != dest);
